docs/tools/developer-tools/lab/api-explorer/horizon-endpoint.mdx

```diff
@@ -0,0 +1,360 @@
+# Horizon Endpoint
+
+Horizon provides an HTTP API to access data stored on the Stellar network. This API acts as a bridge between applications and Stellar Core. Stellar Lab's [API Explorer](https://lab.stellar.org/endpoints) offers a Horizon Endpoint UI, enabling developers to interact with Horizon on Futurenet, Testnet, Mainnet, and custom networks. Developers can use Stellar Lab to build and submit transactions, query account balances, and stream events such as transactions for an account.
+
+:::info
+
+On August 1, 2024, the SDF truncated historical data on its Horizon instances to one year. This update allows us to optimize performance and ensure a streamlined experience for all users. We encourage you to explore [third-party ecosystem providers of Horizon](../../../../data/horizon/horizon-providers), which may provide a longer history retention window as well as other features.
+
+:::
+
+Data on the Stellar ledger is organized into [resources](https://developers.stellar.org/docs/data/horizon/api-reference/resources): **Accounts, Assets, Claimable Balances, Effects, Ledgers, Liquidity Pools, Offers, Operations, Trades**, and **Transactions**. Each resource has multiple endpoints. Endpoints that aggregate ledger data (also known as [aggregations](https://developers.stellar.org/docs/data/horizon/api-reference/aggregations)) include resources like **Order Books, Paths, Trade Aggregations**, and **Fee Stats**.
+
+Stellar Lab's Horizon Endpoint lists each resource type in the side menu, allowing developers to interact with its respective endpoints.
+
+![Lab: Horizon Page](/assets/lab/horizon-endpoints.png)
+
+:::info
+
+For detailed information about each endpoint on a page, click the **View Docs** link in the header to access its API Reference documentation.
+
+:::
+
+![Lab: Horizon - View Docs](/assets/lab/lab-horizon-view-docs.png)
+
+## Examples of Horizon Endpoints
+
+Let's explore a few examples of Horizon endpoints on Testnet and Mainnet. We also have a [documentation on how to set up a local Stellar network environment](../quickstart-with-lab) if you are interested in using a local environment. This will demonstrate how to use the lab to access these endpoints.
```

Comment:
```suggestion
Let's explore a few examples of Horizon endpoints on Testnet and Mainnet. We also have a [documentation on how to set up a local Stellar network environment](../quickstart-with-lab) if you are interested in using a local environment. This will demonstrate how to use the Lab to access these endpoints.
```